fldigi.yml

```diff
@@ -0,0 +1,37 @@
+app-id: org.freedesktop.fldigi
```

Comment:
The file should be renamed to `org.freedesktop.fldigi.yml`

Reply:
ok